clients/js/src/vaa.ts

```diff
@@ -116,7 +118,7 @@ export function parse(buffer: Buffer): VAA<Payload | Other> {
   } else {
     delete (payload as any)["tokenURILength"];
   }
-  var myVAA = { ...vaa, payload };
+  var myVAA = { ...vaa, rawPayload: vaa.payload.toString("hex"), payload };
```

Comment:
I was getting frustrated that the only thing displayed was the parsed payload while the circle integration has a parsing method for the payload and I needed it for the tests. I thought it would be nice to display both.